tests/kafkatest/services/kafka/kafka.py

```diff
@@ -79,6 +79,8 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
         self.sasl_mechanism = sasl_mechanism
         self.topics = topics
         self.minikdc = None
+        if use_authorizer:
+            self.authorizer_class_name = "kafka.security.auth.SimpleAclAuthorizer"
```

Comment:
Is this the only authorizer we'll want to test? It looks like the `use_authorizer` was used as a proxy for the `authorizer_class_name` because the default you want is no authorizer, but it's easier to not have to specify the class name anywhere you're passing `use_authorizer=True`, but the boolean `use_authorizer` seems pretty inflexible. Any reason not to just specify the exact `authorizer_class_name` as the constructor parameter, provide an easy to use constant via the `KafkaService` class (e.g. `KafkaService.SIMPLE_AUTHORIZER`)?